docs/reference/indices/analyze.asciidoc

```diff
@@ -52,3 +52,16 @@ mapping for `obj1.field1` (and if not, the default index analyzer).
 
 Also, the text can be provided as part of the request body, and not as a
 parameter.
+
+And all parameter can be provided as JSON of the request body.
```

Comment:
I think we should promote this usage as the primary usage of the API. Can you change the documentation above? Also add a "comming[2.0.0, body based parameters were added in 2.0.0]" next to each example